lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -429,6 +429,17 @@ sub set_collaborators {
     $self->sql->commit;
 }
 
+sub set_comment {
+    my ($self, $collection_id, $entity_id, $entity_type, $comment) = @_;
+
+    $self->sql->do(<<~"SQL", $comment, $collection_id, $entity_id);
+        UPDATE editor_collection_$entity_type
+           SET comment = ?
+         WHERE collection = ?
+           AND $entity_type = ?
+        SQL
```

Comment:
I understand we validate `$entity_type` in the WS endpoint by checking for its presence in `%ENTITIES`, but since it still comes from an untrusted WS query parameter I'd feel more comfortable if we quoted the identifiers here, just in case the validation fails -- perhaps after a future refactoring, or if we end up using `set_comment` in another location that doesn't validate things properly.
```suggestion
    my $dbh = $self->sql->dbh;
    my $table = $dbh->quote_identifier("editor_collection_$entity_type");
    my $entity_column = $dbh->quote_identifier("$entity_type");
    $self->sql->do(<<~"SQL", $comment, $collection_id, $entity_id);
        UPDATE $table
           SET comment = ?
         WHERE collection = ?
           AND $entity_column = ?
        SQL
```